lib/cisco_node_utils/interface.rb

```diff
@@ -933,7 +933,11 @@ def default_switchport_mode
     end
 
     def switchport_trunk_allowed_vlan
-      config_get('interface', 'switchport_trunk_allowed_vlan', name: @name)
+      vlans = config_get('interface', 'switchport_trunk_allowed_vlan',
+                         name: @name)
+      vlans = vlans.join(',') if vlans.is_a?(Array)
+      vlans = Utils.normalize_range_array(vlans, :string) unless vlans == 'none'
```

Comment:
It is and it isn't. :-(

pvlan correctly uses none:
https://github.com/cisco/cisco-network-node-utils/blob/release_1.3.1/lib/cisco_node_utils/cmd_ref/interface.yaml#L447

non-pvlan is wrong, has a TBD to fix it like pvlan:
https://github.com/cisco/cisco-network-node-utils/blob/release_1.3.1/lib/cisco_node_utils/cmd_ref/interface.yaml#L480